### PR TITLE
Solve #28

### PR DESCRIPTION
**Pull request by avyfain**
This PR solves "Sponsor Page Graph Integration #28".
Charts still need some styling and some work, but you can see them now. Notice the d3 code uses d3 v3, and not d3 v4 which was originally being referenced in the `<script>` tag

Also makes minor wording changes, per conversation with Grace.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -87,3 +87,4 @@ ENV/
 
 # Rope project settings
 .ropeproject
+dev.db
```
Comment (PirosB3) [inline .gitignore]: redundant

File: pybay/templates/frontend/base.html [hunk 2]
```diff
@@ -270,7 +270,11 @@ <h1 class="large-h1">Subscribe to stay informed</h1>
         <script src="{% static 'new/js/twitterfetcher.min.js' %}"></script>
         <script src="{% static 'new/js/placeholders.min.js'%}"></script>
         <script src="{% static 'new/js/scripts.js'%}"></script>
-        <script src="https://d3js.org/d3.v4.min.js"></script>
-        <script src="{% static 'new/js/chart3.js' %}"></script>
+
+		<!-- These should probably just be on the sponsors page -->
+		<script src="https://d3js.org/d3.v3.min.js"></script>
```
Comment (PirosB3) [inline pybay/templates/frontend/base.html]: are we sure we can use this? it's not even a CDN. Suggest to add file locally, or alternatively move to a CDN

**Comment by avyfain** [comment]
@PirosB3, fixed, per comments.

**Comment by avyfain** [comment]
Closing this, please look at #42 instead.

**Pull request closed without merging**